File: benchmarks/accuracy/fixtures/go/control_parameterized_sql.go
package main

import (
	"database/sql"
	"net/http"
)

func findUser(db *sql.DB, w http.ResponseWriter, r *http.Request) {
	name := r.URL.Query().Get("name")
	rows, err := db.Query("SELECT id, email FROM users WHERE name = $1", name)
	if err != nil {
		http.Error(w, "lookup failed", http.StatusInternalServerError)
		return
	}
	defer rows.Close()
}

File: benchmarks/accuracy/fixtures/javascript/control_parameterized_sql.js
const express = require('express');
const db = require('./db');

const router = express.Router();

router.get('/products', (req, res) => {
    const { name } = req.query;
    const sql = 'SELECT id, price FROM products WHERE name = ?';
    db.query(sql, [name], (err, rows) => res.json(rows));
});

module.exports = router;
